python/cudf/cudf/core/column/categorical.py

```diff
@@ -25,25 +26,15 @@ def __init__(self, parent):
     def categories(self):
         from cudf.core.index import as_index
 
-        return as_index(self._parent._categories)
-
-    @property
-    def ordered(self):
-        return self._parent._ordered
+        return as_index(self._parent.categories)
 
     @property
     def codes(self):
-        from cudf import Series
+        return cudf.Series(self._parent.codes)
```

Comment:
Should this maintain name / Index? I saw we at least maintained name previously.

Reply:
Pandas doesn't maintain the name, but it does maintain the index. We should fix to do the same

Reply:
Follow up PR, will open issue.